lib/helpers/stubs.js

```diff
@@ -0,0 +1,125 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+const Immutable = require('immutable')
+const {roundtrip} = require('../util/request')
+
+module.exports = [
+  {
```

Comment:
I would suggest that we change this from array to object and move name from the current object to the root level key. This way you can include the ones that you need in the test. For example in test you define array of keys that you need and then you add then with for loop